timesketch/__init__.py

```diff
@@ -36,6 +36,10 @@
 from timesketch.views.home import home_views
 from timesketch.views.sketch import sketch_views
 from timesketch.views.auth import auth_views
+from timesketch.views.spa import spa_views
+
+# Set to true to use the new Vue.js based frontend.
+USE_NEW_FRONTEND = True
```

Comment:
should this not be set to False by default? Or read from a config file...

Reply:
Good catch! Yes it should definitely be set to False by default :)
Fixing.

Reply:
Fixed